web/modules/custom/dpl_library_agency/dpl_library_agency.module

```diff
@@ -4,3 +4,10 @@
  * @file
  * Contains dpl_library_agency.module.
  */
+
+/**
+ * Implements hook_dpl_protected_nodes_get_protected_nodes().
+ */
+function dpl_library_agency_dpl_protected_nodes_get_protected_nodes(): array {
+  return dpl_protected_nodes_get_context('Drupal\dpl_library_agency\Form\GeneralSettingsForm', 'dpl_library_agency.general_settings');
```

Comment:
Is it possible to refer to the class like this (without quotes)?:
```
Drupal\dpl_library_agency\Form\GeneralSettingsForm::class
```
In that way we can directly see if the class exists instead of using a string as indicator.

Reply:
Implemented suggestion.